todocore/src/main/java/com/knighttodo/todocore/rest/DayTodoResource.java

```diff
@@ -81,7 +81,7 @@ public DayTodoResponseDto addDayTodo(@PathVariable UUID dayId, @Valid @RequestBo
     })
     public List<DayTodoResponseDto> findDayTodosByDayId(@PathVariable UUID dayId) {
         try {
-            return dayTodoService.findByDayId(dayId)
+            return dayTodoService.findByDayOrderNumber(dayId)
```

Comment:
You shouldn't change the name of the method, it's still findByDayId, number is just a new field in the model (and entity and request and response)

Reply:
Done.